ui_framework/src/components/code_editor/__snapshots__/code_editor.test.js.snap

```diff
@@ -0,0 +1,29 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`KuiCodeEditor is rendered 1`] = `
+<div
+  class="kuiCodeEditorWrapper"
+>
+  <div
+    class="kuiCodeEditorKeyboardHint"
+    id="42"
+    role="button"
+    tabindex="0"
+  >
+    <p
+      class="kuiText kuiVerticalRhythmSmall"
+    >
+      Press Enter to start editing.
+    </p>
+    <p
+      class="kuiText kuiVerticalRhythmSmall"
+    >
+      When you’re done, press Escape to stop editing.
+    </p>
+  </div>
+  <div
+    id="brace-editor"
```

Comment:
Do we know why the rest of the editor doesn't get rendered here? Maybe we can use [mount](https://github.com/airbnb/enzyme/blob/master/docs/api/mount.md) in the test instead of `render`, then call the `html()` method on the resulting object, and store that string as the snapshot?

Reply:
Done